src/app.js

```diff
@@ -1 +1,31 @@
+/* eslint-disable no-console */
 'use strict';
+
+const args = process.argv.slice(2);
+
+const operation = args[0];
+
+const numbers = args.slice(1).filter(arg => !isNaN(arg)).map(Number);
+
+if (numbers.length === 0) {
+  if (operation === 'add') {
+    console.log('There is nothing to add.');
+  } else if (operation === 'multiply') {
+    console.log('There is nothing to multiply.');
```

Comment:
You can combine the logic for multiply and add
```suggestion
    console.log(`There is nothing to ${operation}.`);
```